data-service/src/main/java/org/ihtsdo/buildcloud/service/worker/SRSWorkerService.java

```diff
@@ -0,0 +1,116 @@
+package org.ihtsdo.buildcloud.service.worker;
+
+import org.ihtsdo.buildcloud.entity.Build;
+import org.ihtsdo.buildcloud.entity.QATestConfig;
+import org.ihtsdo.buildcloud.service.BuildService;
+import org.ihtsdo.otf.jms.MessagingHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.web.authentication.preauth.PreAuthenticatedAuthenticationToken;
+import org.springframework.stereotype.Service;
+import org.springframework.util.Assert;
+
+import javax.jms.JMSException;
+import javax.jms.TextMessage;
+
+import static org.ihtsdo.buildcloud.service.build.RF2Constants.*;
+import static org.ihtsdo.otf.jms.MessagingHelper.AUTHENTICATION_TOKEN;
+
+@Service
+public class SRSWorkerService {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(SRSWorkerService.class);
+
+	@Autowired
+	private BuildService buildService;
+
+	@Autowired
+	private MessagingHelper messagingHelper;
+
+	@JmsListener(destination = "${srs.jms.job.queue}")
+	public void consumeSRSJob(final TextMessage srsMessage) {
+		LOGGER.info("Product build request message {}", srsMessage);
```

Comment:
The job message needs to be updated to include config from GatherInputRequestPojo